ChemoTracker/App.js

```diff
@@ -1,24 +1,45 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Platform } from 'react-native';
+import { NavigationComponent } from 'react-native-material-bottom-navigation';
+import { TabNavigator } from 'react-navigation';
 import Home from './components/Home.js';
+import Track from './components/Track.js';
+import Profile from './components/Profile.js';
+import Search from './components/Search.js';
+import Calendar from './components/Calendar.js';
 
-class App extends React.Component {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Home />
-      </View>
-    );
+const App = TabNavigator({
```

Comment:
I don't think its a good idea to have this code here. maybe create a new file and move the logic there so its easier to switch screens to this view. the first thing that will be seen is not supposed to be this screen.

Lets try to leave app.js like a controller and have different components in their respective files.

Reply:
This PR is intended just to add navigation, that's why the navigation is inside the app directly.. I searched it up online https://engineering.musefind.com/getting-started-with-react-native-in-20-minutes-15ea90062094 , although I havent tested it, I will separate the navigation into a different file. Since you're working on the login and the home screen, I think you can refer to the link to change the App.js to link to Login/Home in the future after this PR is merged?

Reply:
yeah sure.